recipes/configs/llama2/7B_full.yaml

```diff
@@ -3,7 +3,7 @@
 #
 # This config assumes that you've run the following command before launching
 # this run:
-#   tune download meta-llama/Llama-2-7b --hf-token <HF_TOKEN> --output-dir /tmp/llama2
+#   tune download meta-llama/Llama-2-7b-hf --output-dir /tmp/Llama-2-7b-hf --hf-token <TOKEN>
```

Comment:
Unless there's a good reason to change it can we keep <HF_TOKEN> instead of <TOKEN>? That's how we have it everywhere else